### PR TITLE
Rgo2019 challenge where is this

### LINKED ISSUE #816: RGO2019 PR tue_robocup rgo2019_challenge_where_is_this

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Challenge is actually broken at the moment by the CONFIRM_SAY state.

EDIT: The breaking commit, has been reverted. Ready for reviewing.

Fixes #816

### CODE REVIEW

File: challenge_where_is_this/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Storing groceries
```
Comment (PetervDooren) [inline challenge_where_is_this/README.md]: This is the README of storing groceries. Please update the README with the correct challenge description and instructions

File: robocup_knowledge/src/robocup_knowledge/environments/rgo2019/challenge_where_is_this.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+location_grammar = """
+T[A] -> COURTESY_PREFIX VP[A] | VP[A]
+
+COURTESY_PREFIX -> robot please | could you | could you please | please
+"""
+
+for loc in common.location_rooms:
+    location_grammar += '\nLOCATION[{"id": "%s"}] -> %s' % (loc, loc)
+
+for loc in common.location_names:
```
Comment (PetervDooren) [inline robocup_knowledge/src/robocup_knowledge/environments/rgo2019/challenge_where_is_this.py]: These lines are double

Comment (JosjaG) [reply robocup_knowledge/src/robocup_knowledge/environments/rgo2019/challenge_where_is_this.py]: I do think there's a difference between ```location_rooms``` and ```location_names``` ;)

**Comment by JosjaG** [comment]
Adding comments is definitely important but this should be done by @jlunenburg or @MatthijsBurgh since they wrote this, and they are currently busy. So let's leave this for now.

File: robocup_knowledge/src/robocup_knowledge/environments/rgo2019/challenge_where_is_this.py (hunk 2 above)
Comment (PetervDooren) [reply robocup_knowledge/src/robocup_knowledge/environments/rgo2019/challenge_where_is_this.py]: I will have my glasses checked ;P

**Pull request merged**